test/integration/global_deploy_test.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+require 'integration_test_helper'
+
+class GlobalDeployTest < Krane::IntegrationTest
+  def test_global_deploy_task_success
+    assert_deploy_success(deploy_global_fixtures('globals'))
+
+    assert_logs_match_all([
+      "Phase 1: Initializing deploy",
+      "Using resource selector test=",
+      "All required parameters and files are present",
+      "Discovering resources:",
+      "  - StorageClass/testing-storage-class",
+      "Phase 2: Checking initial resource statuses",
+      %r{StorageClass\/testing-storage-class[\w-]+\s+Not Found},
+      "Phase 3: Deploying all resources",
+      %r{Deploying StorageClass\/testing-storage-class[\w-]+ \(timeout: 300s\)},
+      "Don't know how to monitor resources of type StorageClass.",
+      %r{Assuming StorageClass\/testing-storage-class[\w-]+ deployed successfully.},
+      %r{Successfully deployed in 0.[\d]+s: StorageClass\/testing-storage-class},
```

Comment:
Oh, this needs to be relaxed:
> Successfully deployed in 0.[\d]+s: StorageClass\/testing-storage-class)' not found in the following logs:
Because it took more than a second for whatever reason